domains/atharvd.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "D-Atharv",
+        "email": "atharv.dewangan04@gmail.com"
+    },
+    "record": {
+        "CNAME": "my-portfolio-dusky-six-78.vercel.app"
```

Comment:
You should complete your website first, i only see a "Coming Soon" text

Reply:
Hoping to hear soon. thanks